src/main/java/com/epam/emotionalhelp/controller/dto/EmotionRequestDto.java

```diff
@@ -2,13 +2,11 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.Value;
 
-@Data
+@Value
 @AllArgsConstructor
```

Comment:
There is no need to use `@AllArgsConstructor` annotation if we have `@Value` annotation. And we do not need a builder only for a one field.